Misc/NEWS.d/next/Core and Builtins/2020-02-11-15-54-40.bpo-39610.fvgsCl.rst

```diff
@@ -0,0 +1,2 @@
+``len()`` for 0-dimensional ``memoryview`` objects (such as ``memoryview(ctypes.c_uint8(42))``) now raises a ``TypeError``.
+Previously this returned ``1``, which was not consistent with ``mem_0d[0]`` raising an ``IndexError``.

```

Comment:
Presumably there should be a `:class:` or `:type:` or `:exc:` in here somewhere; can a reviewer advise what the convention is?

Reply:
Done